webrender/res/ps_border_corner.fs.glsl

```diff
@@ -78,32 +89,11 @@ void main(void) {
     // necessary for correctness when the border width is greater
     // than the border radius.
     if (all(lessThan(local_pos * vClipSign, vClipCenter * vClipSign))) {
-        float d0, d1;
-
-        // sdEllipse fails on exact circles, so handle equal
-        // radii here. The branch coherency should make this
-        // a performance win for the circle case too.
-        if (vOuterRadii.x == vOuterRadii.y) {
-            // Get the distances to the inner and outer radii.
-            d0 = distance(vClipCenter, local_pos) - vOuterRadii.x;
-        } else {
-            // Get the distance to the outer ellipse.
-            vec2 p = local_pos - vClipCenter;
-            d0 = sdEllipse(p, vOuterRadii);
-        }
-
-        if (vInnerRadii.x == vInnerRadii.y) {
-            d1 = distance(vClipCenter, local_pos) - vInnerRadii.x;
-        } else {
-            // Get distance to inner ellipse. Skip if the inner
-            // radius is <= 0.0 to avoid FP errors.
-            if (vInnerRadii.x <= 0.0 || vInnerRadii.y <= 0.0) {
```

Comment:
are we missing this check with the new code?
Edit: I don't think we ever hit it. `vInnerRadii` is either both zeroes, or both positive. In the former case it will go the `vInnerRadii.x == vInnerRadii.y` path, so this branch will never hit (with either new or old code).